src/foam/mlang/mlang.js

```diff
@@ -2323,11 +2323,49 @@ foam.CLASS({
     }
   ],
 
+  properties: [
+    {
+      class: 'Boolean',
+      name: '_checkingNestedFObject',
```

Comment:
Can you rename this checkingNextedFObject_?

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/3796/commits/15fbc8b03b41b76ca19e4ed4e9e41fa3a0ccaaf3.